python/py-eyed3/Portfile

```diff
@@ -15,21 +15,22 @@ license             GPL-3+
 maintainers         nomaintainer
 
 description         Python audio data toolkit (ID3 and MP3)
-long_description    eyeD3 is a Python program/module for processing \
-                    (reading and writing) ID3 tags. Information about mp3 \
-                    files (i.e bit rate, sample frequency, play time, \
-                    etc.) is also available. The formats supported are ID3 \
-                    v1.0/v1.1 and v2.3/v2.4.
+long_description    eyeD3 is a Python tool for working with audio files, specifically MP3 files \
+                    containing ID3 metadata (i.e. song info). \
+                    It provides a command-line tool (eyeD3) and a Python library (import eyed3) \
+                    that can be used to write your own applications or plugins that are callable \
+                    from the command-line tool.
 
 homepage            https://eyed3.readthedocs.io
 
-checksums           rmd160  e021b13c5f031d54e6b17af94e948e1e568ac40c \
-                    sha256  11099464e438c11a1d701e723a5065c1556fb59878ad9dce29f924dac3a07a96 \
-                    size    222818
+checksums           rmd160  68444eee7f0c2bf95c0feb7ee779ad27c8a88fff \
+                    sha256  4b5064ec0fb3999294cca0020d4a27ffe4f29149e8292fdf7b2de9b9cabb7518 \
+                    size    221831
 
-python.versions     27 37 38
+python.versions     27 36 37 38 39
```

Comment:
please don't add a subport for PY36 unless you actually want to use it (if that's the case, you will need to add the PY36 subports to dependents as well). By now Python 3.6 is quite old already and every subport we add now will need to be removed again at some point.

Reply:
makes sense. will do.